### ISSUE TITLE
[byu-search] Add example of `input` event to documentation

### ISSUE THREAD

**Issue by PawnStar** [byuweb/byu-theme-components#305]
# Description

It would be very nice if there was some way to be notified when the search input value has been changed, so external scripts could use that information (such as for AJAX suggestion).  Ideally this would be by registering a callback with the element, but it would even be nice if the value was just available for polling.

# Websites Affected

It's not really a bug, but we'd like the feature for https://docs.physics.byu.edu/

# Issue Type

*Is this (add an x in the boxes that apply)*

- [ ] A difference between the components and the [Official Specification](https://brand.byu.edu)?
- [ ] A bug, such as a Javascript error, or the UI not rendering properly
    on a page?
- [ ] Inconsistent appearance/behavior between browsers?
- [ ] An issue on mobile browsers?
- [x] A request for a new feature/enhancement?

# Browsers Affected

Not applicable (not a bug), but I guess you could say "all of them" because it's not implemented in any.

# Web Site Platform

*What is hosting your website?*

- [ ] Drupal 7
- [ ] Drupal 8
- [ ] Wordpress
- [x] Custom Site (Dokuwiki)
- [ ] Don't Know

**Comment by ThatJoeMoore** [comment]
This is actually already possible using the standard `input` event:

```
<byu-search>
  <input oninput="searchChanged()"/>
</byu-search>
```

I don't think we should provide any other way to access this functionality than that already provided us by the Web Platform.

**Comment by ThatJoeMoore** [comment]
Hmm...We should make a note of this in the documentation, though.  I'm going to re-open this to track such a documentation change.

**Comment by PawnStar** [comment]
I don't know why I didn't think of that. Thanks for pointing me in the right direction.

**Comment by blakemorgan** [comment]
Fixed in #375
